wen-restart/src/last_voted_fork_slots_aggregate.rs

```diff
@@ -16,16 +16,26 @@ use {
     },
 };
 
+// If at least 1/3 of the stake has voted for a slot in next Epoch, we think
+// the cluster's clock is in sync and everyone will enter the new Epoch soon.
+// So we require that we have >80% stake in the new Epoch to exit.
+const EPOCH_CONSIDERED_FOR_EXIT_THRESHOLD: f64 = 33.33;
```

Comment:
Should we keep this consistent as a threshold `const EPOCH_CONSIDERED_FOR_EXIT_THRESHOLD: f64 = 1f64 / 3f64` nstead of a percent? Seems more in line with all the other thresholds we have

Reply:
Done.